generators/typescript/utils/commons/src/typescript-project/BundledTypescriptProject.ts

```diff
@@ -313,6 +320,17 @@ export * from "./${BundledTypescriptProject.TYPES_DIRECTORY}/${folder}";
         await this.writeFileToVolume(RelativeFilePath.of("package.json"), JSON.stringify(packageJson, undefined, 4));
     }
 
+    private async generateJsrJson(): Promise<void> {
+        if (this.npmPackage != null) {
+            const jsr: JSR = {
+                name: this.npmPackage?.packageName,
+                version: this.npmPackage.version,
+                exports: "src/index.ts"
```

Comment:
Dumb q: do we not output index.ts if we're in esm mode? Like instead we do index.js + index.d.ts?

Reply:
We actually did for Hookdeck when we were in `esmMode`

Reply:
I think we dont output it on local mode